plasma_cash/client/child_chain_client.py

```diff
@@ -1,15 +1,34 @@
+import json
+import threading
+
 import requests
+import websocket
 
 from .exceptions import RequestFailedException
 
 
 class ChildChainClient(object):
 
-    def __init__(self, base_url, verify=False, timeout=5):
+    def __init__(self, base_url, ws_url, verify=False, timeout=5):
         self.base_url = base_url
         self.verify = verify
         self.timeout = timeout
 
+        self.ws = websocket.WebSocketApp(ws_url, on_message=self.ws_on_message)
+        self.ws_callback = {}
+        threading.Thread(target=self.ws.run_forever).start()
+
+    def ws_on_message(self, ws, message):
```

Comment:
if possible please add unit test for these new functions as well

Reply:
sure.